gradle-plugin/doc/src/main/asciidoc/inc/scenarios/_quarkus.adoc

```diff
@@ -0,0 +1,74 @@
+[[quarkus-scenario]]
+== Quarkus
+JKube provides ready to use generators. A JKube generator is composed of default configurations to build container images, generate k8s resources and deploy them to kubernetes clusters. There are predifined generators for each type of Java project. For instance, we have a generator for Quarkus projects. It helps JKube to identify that a project is a Quarkus one and provides the right configuration to be used.
+
+This is an example of how you can use the JKube zero configuration for a Quarkus project.
+
+[[quarkus-scenario-prerequisites]]
+=== Prerequisites
+You will need the following for the scenario:
+
+* `minikube` installed and running on your computer
```

Comment:
I just realized this same documentation would be used for OpenShift Gradle Plugin. Do you think we should show this only for Kubernetes Gradle Plugin?

```adoc
ifeval::["{task-prefix}" == "k8s"]

// scenario on minikube 

endif::[]

ifeval::["{task-prefix}" == "oc"]

// scenario on CRC/OpenShift Cluster

endif::[]
```